### PR TITLE
Serve static files.

### PR DESCRIPTION
**Pull request by mindhog**
Serve static files through devappserver and expose the "proxy port" so the users can get to it.

Fixes issues/18

@rbuskens please review?

### CODE REVIEW

File: appstart/sandbox/container_sandbox.py [hunk 1]
```diff
@@ -464,8 +467,19 @@ def exit_loop_with_error(error):
             attempt += 1
             time.sleep(1)
 
+        # Tell the user where to connect, depending on whether the
```
Comment (rbuskens) [inline appstart/sandbox/container_sandbox.py]: 'whether' --> 'whether or not'

File: appstart/sandbox/container_sandbox.py [hunk 2]
```diff
@@ -464,8 +467,19 @@ def exit_loop_with_error(error):
...
+        # devappserver is running.
+        if self.run_devappserver:
+            port = self.proxy_port
+        else:
+            port = self.port
         get_logger().info('Your application is live. '
-                          'Access it at: {0}:{1}'.format(host, str(self.port)))
+                          'Access it at: {0}:{1}'.format(host, port))
+        if self.run_devappserver:
+            get_logger().info('(port {0} goes through the dev_appserver '
```
Comment (rbuskens) [inline appstart/sandbox/container_sandbox.py:479]: Is there any good reason not to move this under line 472/473?

File: appstart/sandbox/container_sandbox.py [hunk 3]
```diff
@@ -507,18 +521,21 @@ def build_devappserver_image(self,devbase_image=constants.DEVAPPSERVER_IMAGE):
...
-        ADD %(path)s/* %(dest)s
+        ADD %(path)s/ %(dest)s
+        WORKDIR /app/
         """ % {'das_repo': devbase_image,
                'path': os.path.dirname(self.conf_path),
                'dest': os.path.join('/app', self.das_offset)}
 
         # Construct a file-like object from the Dockerfile.
         dockerfile_obj = io.BytesIO(dockerfile.encode('utf-8'))
+        print 'files to add = %s' % files_to_add
```
Comment (rbuskens) [inline appstart/sandbox/container_sandbox.py]: Is this for debugging purposes? Should get_logger() be used instead?

File: appstart/utils.py [hunk 4]
```diff
@@ -251,6 +252,36 @@ def make_tar_build_context(dockerfile, context_files):
...
+    handlers = config.get('handlers')
+    if handlers and isinstance(handlers, list):
+        for handler in handlers:
+            if not isinstance(handler, dict):
+                raise AppstartAbort('"handlers" section of {!r} contains an '
+                                    'illegal value'.format(config_name))
+            static_dir = handler.get('static_dir')
+            if static_dir:
+                if not isinstance(static_dir, basestring):
+                    raise AppstartAbort('"handlers" section of {!r} contains an '
```
Comment (rbuskens) [inline appstart/utils.py]: 'an' --> 'a'

**Comment by rbuskens** [comment]
LGTM, with minor comments.

File: appstart/sandbox/container_sandbox.py (hunk 1 above)
Comment (mindhog) [inline appstart/sandbox/container_sandbox.py]: done

File: appstart/sandbox/container_sandbox.py (hunk 2 above)
Comment (mindhog) [inline appstart/sandbox/container_sandbox.py:479]: It makes for a more user-friendly message flow without duplicating the log message.

File: appstart/sandbox/container_sandbox.py (hunk 3 above)
Comment (mindhog) [inline appstart/sandbox/container_sandbox.py]: Yep.  Removed.

File: appstart/utils.py (hunk 4 above)
Comment (mindhog) [inline appstart/utils.py]: done

**Pull request merged**